spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -1159,4 +1160,72 @@ Spring Cloud Data Flow will have a suffix appended to it consisting of `-scdf-<t
 For example if I create a schedule name of `my-schedule` for a task definition of `tstamp` the
 schedule name will look like `my-schedule-scdf-tstamp`.
 
-NOTE: The `scdf-` separator is configurable by setting the `spring.cloud.dataflow.task.scheduleNamePrefix` property with the value you prefer.
\ No newline at end of file
+NOTE: The `scdf-` separator is configurable by setting the `spring.cloud.dataflow.task.scheduleNamePrefix` property with the value you prefer.
+
+[[spring-cloud-dataflow-task-cd]]
+== Continuous Deployment
+As task applications evolve, you will want to get your updates to production.  This section will walk through the capabilities that Spring Cloud Data Flow provides around being able to update task applications.
+
+When a task application is registered (<<spring-cloud-dataflow-register-task-apps>>), there is a version associated with it.  A task application can have multiple versions associated with it with one selected as the default.
+
+image::{dataflow-asciidoc}/images/dataflow-task-application-versions.png[Task Application Versions, scaledwidth="50%"]
```

Comment:
Images should have leader sentences: "The following image shows ..." or "This image shows ...". It can be its own paragraph if it doesn't fit well with the preceding paragraph.